modules/cluster-logging-release-notes-5.4.0.adoc

```diff
@@ -50,6 +50,9 @@ link:https://access.redhat.com/errata/RHSA-2022:1461[{logging-title-uc} Release
 
 * Before this update, a change to the container logs path caused this metric to always be zero with older releases configured with the original path. With this update, the plugin which exposes metrics about collected logs supports reading from either path to resolve the issue. (link:https://issues.redhat.com/browse/LOG-2462[LOG-2462])
 
+== Elasticsearch Operator deprecation notice
+In {logging} 5.4, the Elasticsearch Operator is deprecated and is expected to be removed for the next EUS release of {product-title}.  Red Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed. As an alternative to using the Elasticsearch Operator to manage the default log storage, you can use the Loki Operator.
```

Comment:
This looks good to me but we really need complete agreement from Jon, Shannon, and @periklis , and anyone else from program management